src/Microsoft.OData.Core/JsonLight/ODataJsonLightParameterDeserializer.cs

```diff
@@ -25,6 +26,11 @@ internal sealed class ODataJsonLightParameterDeserializer : ODataJsonLightProper
         private static readonly Func<string, object> propertyAnnotationValueReader =
             annotationName => { throw new ODataException(ODataErrorStrings.ODataJsonLightParameterDeserializer_PropertyAnnotationForParameters); };
 
+        /// <summary>OData property annotation asynchronous reader for parameter payloads.</summary>

```

Comment:
I don't think this statement is necessary here. Its contradicting the second statement. You can get rid of this and just leave the second one.

Reply:
Addressed by moving the statement about `ODataJsonLightParameterDeserializer` not supporting reading of property annotations to the body of the delegate